pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -315,77 +325,58 @@ func CreatePoller(
 	return p
 }
 
-// Start the poller workers
+// Start the poller workers and libvirt API operations
 func (p *AgentPoller) Start() {
 	if p.agentDone != nil {
 		return
 	}
 	p.agentDone = make(chan struct{})
 
 	for i := 0; i < len(p.workers); i++ {
-		log.Log.Infof("Starting agent poller with commands: %v", p.workers[i].AgentCommands)
 		go p.workers[i].Poll(func(commands []AgentCommand) {
 			executeAgentCommands(commands, p.Connection, p.agentStore, p.domainName)
 		}, p.agentDone, pollInitialInterval)
 	}
+
+	sysApiCallTick := 120 * time.Second
```

Comment:
I'd simply keep the flags. I don't see why we need to drop them.

Reply:
Do you mean to keep CLI flags?

Reply:
Yes